WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -764,6 +777,16 @@ - (UITableViewCell *)configureGeolocationCellForIndexPath:(NSIndexPath *)indexPa
     return cell;
 }
 
+- (UITableViewCell *)configureExcerptCellForIndexPath:(NSIndexPath *)indexPath
+{
+    UITableViewCell *cell = [self getWPTableViewCell];
```

Comment:
Actually, if it were called `newWPTableViewCell` the compiler would assume that it's a retained object, but `getWPTableViewCell` sometimes calls `init` and sometimes it `dequeues` an existing cell. I haven't thought about memory management in a long time, but I have the feeling that `new*` would break something 😬 

Honestly, the whole class calls for a refresh, so I would limit this PR to its initial scope.